### PR TITLE
test: add test cases about CRI image

### PR DESCRIPTION
**Pull request by Starnop**
Signed-off-by: Starnop <starnop@163.com>

### Ⅱ. Does this pull request fix one issue?

fixes part of #1756

### CODE REVIEW

**Comment by allencloud** [comment]
CI fails:
```
gometalinter
gometalinter --config .gometalinter.json ./...
cri/v1alpha1/cri_utils_test.go:1::warning: file is not gofmted with -s (gofmt)
cri/v1alpha1/cri_utils_test.go:1::warning: file is not goimported (goimports)
Makefile:122: recipe for target 'gometalinter' failed
make: *** [gometalinter] Error 1
```
@Starnop

**Comment by allencloud** [comment]
any update and review? @YaoZengzeng

File: cri/v1alpha2/cri_utils_test.go [hunk 1]
```diff
@@ -785,7 +788,39 @@ func Test_imageToCriImage(t *testing.T) {
...
+			want: &runtime.Image{
+				Id:          "image-id",
+				RepoTags:    repoDigests,
+				RepoDigests: repoDigests,
+				Size_:       uint64(1024),
+				Uid:         &runtime.Int64Value{},
+				Username:    "foo",
+				Volumes:     parseVolumesFromPouch(map[string]interface{}{"foo": "foo"}),
+			},
+			wantErr: nil,
```
Comment (YaoZengzeng) [inline cri/v1alpha2/cri_utils_test.go]: The type of `wantErr` is `bool`, we could not assign nil to it.

CI has found this mistake but not failed.

File: cri/v1alpha2/cri_utils_test.go [hunk 2]
```diff
@@ -841,7 +878,30 @@ func Test_getUserFromImageUser(t *testing.T) {
...
 		want1 string
 	}{
-	// TODO: Add test cases.
+		{
+			name: "Empty Test",
+			args: args{
+				imageUser: "",
+			},
+			want:  nil,
+			want1: "",
```
Comment (YaoZengzeng) [inline cri/v1alpha2/cri_utils_test.go]: Use a better name, other than `want1` :)

File: cri/v1alpha2/cri_utils_test.go [hunk 3]
```diff
@@ -841,7 +878,30 @@ func Test_getUserFromImageUser(t *testing.T) {
 		want  *int64
 		want1 string
 	}{
-	// TODO: Add test cases.
+		{
+			name: "Empty Test",
+			args: args{
+				imageUser: "",
+			},
+			want:  nil,
```
Comment (YaoZengzeng) [inline cri/v1alpha2/cri_utils_test.go]: We should check whether the value is equal other then the pointer.

Comment (Starnop) [reply cri/v1alpha2/cri_utils_test.go]: Thanks for your comment. All above mentioned has been done.

**Comment by YaoZengzeng** [comment]
LGTM

**Pull request merged**